docker-compose.yml

```diff
@@ -8,20 +8,6 @@ services:
     working_dir: /Users/samjohnson/Documents/hrfiles/petco/description_directions_attributes_
     ports:
       - "80:3002"
-    links:
-      - "redis"
-    command: bash -c "wait-for-it.sh redis:6379 && npm install && npm run pro"
```

Comment:
Why did you remove the wait-for-it? I ended up not using it.

Reply:
i don't have to wait for redis to connect because i'm not using redis on my service instances anymore